rainbow-me/rainbow
# feat: multimedia NFTs

### Description
This branch adds two capabilities:
  - The ability to detect whether an asset has a valid 3D `.glb` `animation_url`.
    - Unique assets which are detected in this format are represented alongside an additional icon in the `<AssetRecylerList />`.
  - The ability to render a 3D Model using [`<model-viewer/>`](https://modelviewer.dev/).
    - If the asset can be loaded successfully, it will be rendered.
    - If it possesses animation parameters, it will by default `autoplay` the animation.
    - Orbital controls are enabled to allow spherical rotation about the rendered scene.

### Pending Tasks
- Slick Rainbow design influence w.r.t displaying information that an asset has 3d properties. (I'm currently just using a placeholder for this).

### Testing
1. Watch address `0x3b24804587a841c11b4e5baac8a6211671672ce0`
2. Navigate to `SuperRare` collectibles.
3. Tap on Bernie Sanders and watch him shiver. Zoom in closely to check out his comfy gloves.
